Strasbourg/BNUet5.xml

```diff
@@ -0,0 +1,189 @@
+                                ወሀደገ፡ ግብረ። ለድኩማን። ወእምቅድመ፡ ይትፈጸም፡ ፲ወ፩ ዓመተ፡ መንግሥቱ፡ ተከሥተ፡ ክሕደተሰ። ኀበ፡ ምድረ፡ ግብፅ፡ ወተፃብኦ፡ ምስሌሆሙ። ወምኦሙ።
+                                ወጸሐፈ፡ ለብንያም፡ ሊቀ፡ ጳጳስት፡ መጽሐፈ፡ ኪዳን። ወመጽአ፡ ብንያም፡ ኀበ፡ እስክንድርያ፡ እምድኅረ፡ ጐዩ፡ እመንበሩ፡ ፲ወ፩ ዓመት። ወእምድኅረ፡ ዝንቱ፡ 
+                                ፈለሰ፡ ታሪክ፡ ዝንቱ፡ ኆልቈ፡ ዓመታት፡ ወዜናሆሙ፡ ለነገሥት፡ ወካልኦሙ፡ ኀበ፡ ትንባላት። ወኮነ፡ ምጽአቶሙ፡ ለተንባላት፡ ኀበ፡ ምድረ፡ ግብፅ፡ በተፍጻሜተ፡ ፲ወ፩ 
+                                ዓመት፡ እመንግሥተ፡ ህርቃል፡ ወይእቲ፡ ቀዳሚት፡ ዓመት፡ እምዓመታት፡ ተንባላት፡ ወበ፫፻፶ወ፯ ዓመት፡ ለዲዮ፡ ቅልጥያኖስ። ወበ፬፻፴ወ፫ ዓመቱ፡ ለእስክንድር፡
+                                ወበ፯፻፲ወ፬ ዓመት፡ እምልደተ፡ እግዚእነ፡ እየሱስ፡ ክርስቶስ፡ ሎቱ፡ ስብሐት። ወበተፍጻሜተ፡ ፰፻ወ፩፻፲ወ፬ ዓመት፡ እምፍጥረተ፡ ዓለም። ። ። ።</explicit>
+                            <colophon xml:id="coloph1">
+                                <locus from="250v39" to="251r4"/>
+                                <foreign xml:lang="gez">ለገብርከ፡ <persName ref="PRS6064Krapf" role="owner">ክራጵፍ። ዮሐንስ።</persName> ለሊሰ። 
+                                    ዝመጽሐፍ፡ ዘ<persName ref="PRS6064Krapf" role="owner">ክራጵፍ።</persName> ወለጸሐፊሁ፡ 
+                                    <persName ref="PRS0000000000000000000000" role="scribe">ሀብተ፡ ሚካኤል።</persName>ፋጡላ፡ ተፈጸመ፡ 
+                                    <choice resp="CH"><sic>ረ</sic><corr>ለ</corr></choice>ዝንቱ፡ መጽሐፈ፡ ታሪክ፡ ዘአስተጋብአ፡ ብእሲ፡ ክቡር፡ ወትሩፍ፡ 
+                                    ክርስቲያናዊ። ኦርቶዶክሳዊ። ጊዮርጊስ፡ ግብፃዊ። ወኮነ፡ ተፍጻሚተ፡ ውእቱ፡ መጽሐፈ፡ ታሪክ፡ ዘተጽሕፈ፡ እምኔሁ፡ ዝንቱ፡ መጽሐፍ፡ ለአመ፡ ፩ ለወርኃ፡ ግንቦት፡
+                                    በ፻፲ወ፲ወ፩ ዓመት፡ እምሰማዕታት፡ ኅሩያን፡ ወአመ፡ ፯ ለወርኀ፡ ሰዋል። ወበ፯፻፺ወ፫ እምሰማዕታተ፡ ተንባላት። ወዝንቱ፡ መጽሐፍ፡ ዘተጽሐፈ፡ እምብዙኃት፡ 
+                                    መጻሕፍት፡ ዘያ<add place="inline"><unclear>ዐ</unclear></add>ድዕ፡ ኵሎ፡ ነገረ፡ ኅቡአት። ሰብሐት፡ ለእእግዚአብሔር፡ ባበሬ፡ 
+                                    መላእክት፡ አበ፡ ኵሉ፡ ሠርጐ፡ ፍጥረታት፡ ለዓለመ፡ ዓለም፡ አሜን። ።</foreign>
+                                <note><bibl><ptr target="bm:Zanutto1932Bibliografia"/><citedRange unit="number">65</citedRange>
+                                    <citedRange unit="page">43</citedRange></bibl> reports the date of origin of the original: 
+                                    'On the right-hand side of the last folio, a note was added stating that this manuscript was copied by Habta Mikâʼel 
+                                    on the instructions of Krapf, based on a manuscript from the 15th century.'
+                                    <bibl><ptr target="bm:Ewald1844Tuebingen"/><citedRange unit="page">200-201</citedRange></bibl> 
+                                    writes: ‘According to the signature, the manuscript of Elmakin's works, from which the present one is taken, was 
+                                    completed in the year 1010 (!) of the Diocletian era and 765 (!) of the Arabic era, and is therefore relatively very 
+                                    old.'
+                                    However, the claim, that the Vorlage was a manuscript from the 15th century is doubtful. If it were true, this 
```

Comment:
antigraph (a manuscript from which a copy is made) seems to be the correct term to use